x-pack/plugins/security_solution/public/timelines/components/timeline/eql_tab_content/index.tsx

```diff
@@ -282,13 +282,13 @@ export const EqlTabContentComponent: React.FC<Props> = ({
                   setFullScreen={setTimelineFullScreen}
                 />
               )}
-              <DatePicker grow={1}>
+              <DatePicker grow={10}>
```

Comment:
This was a design request i snuck in here so the width of the date picker doesn't jump around when badges are added/removed from the sourcerer